netty/src/test/java/io/grpc/netty/NettyClientHandlerTest.java

```diff
@@ -136,12 +139,31 @@ public class NettyClientHandlerTest extends NettyHandlerTestBase<NettyClientHand
   private ClientStreamListener streamListener;
 
   private final Queue<InputStream> streamListenerMessageQueue = new LinkedList<>();
+  private NettyClientStream stream;
 
   @Override
   protected void manualSetUp() throws Exception {
     setUp();
   }
 
+  @Override
+  protected AbstractStream stream() {
+    if (stream == null) {
+      stream = new NettyClientStream(streamTransportState,
```

Comment:
This needs a call to `stream.start()` to actually create the stream. I'm a bit worried by how the test works without it. (Not a problem with your test in particular, but just the overall pre-existing test structure.)

Reply:
The Netty streams don't have a start method.

Reply:
They most definitely do, because it is [part of the `ClientStream` API](https://github.com/grpc/grpc-java/blob/f5e8459748735c794932637d932070f302779d5e/core/src/main/java/io/grpc/internal/ClientStream.java#L79). The method is implemented on `AbstractClientStream`.

Reply:
You're right that it is only working because stream is never null when this is called.

Reply:
Created PR 9774 to eliminate this unused and incomplete logic.